stripe/http_client.py

```diff
@@ -411,8 +412,8 @@ def __init__(self, verify_ssl_certs=True, proxy=None):
         if self._proxy:
             # now that we have the parser, get the proxy url pieces
             proxy = self._proxy
-            for scheme in proxy:
-                proxy[scheme] = urlparse(proxy[scheme])
+            for scheme, value in proxy.items():
```

Comment:
Nit: can you use `six.iteritems(proxy)` instead? It's slightly more efficient on Python 2 and is consistent with how we iterate over dictionaries in the rest of the library.

Reply:
Sure. I'll make changes.